ClinicalTrialXMLREADME.md

```diff
@@ -0,0 +1,121 @@
+* **sponsors**
+	* **lead_sponsor**
+		* **agency** - organization running the clinical trial
+		* **agency_class** - the type of the organization
+			* Values: NIH, US Fed, Industry, Other
+	* **collaborator**
+		* **agency** - organization supporting the clinical trial that is not the sponsor
+		* **agency_class** - the type of the organization
+			* Values: NIH, US Fed, Industry, Other
+* **source** - same as lead_sponsor agency
+* **oversight_info**
+	* **has_dmc** - whether or not the study has a data monitoring committee
+	* **is\_fda\_regulated_drug** - yes or no
+	* **is\_fda\_regulated_device** - yes or no
+	* **is\_unapproved\_device** - yes or no
+	* **is_ppsd** - ???
+	* **is\_us\_export** - yes or no
+* **brief_summary**
+	* **textblock** - short explaination of what the study is doing and looking to test
+* **detailed_description**
+	* **textblock** - more indepth explanation of the study, includes a lot more information than the breif summary
+* **overall\_status** - potential values depend on the study_type
+	* Study Type: Expanded Access - Values: Available, No Longer Available, Temporarily Not Available, Approved for Marketing
+	* Study Type: N/A - Values: Withheld
+	* Study Type: Anythig Else - Values: Active not recruiting, Completed, Enrolling by initiative, Not yet recruiting, Recruiting, Suspended, Terminated, Withdrawn
+* **last\_known\_status** - most recent status of a study that has not been verified in at least 2 years
+* **why_stopped** - reason for discontinuing the study
+* **start_date**
+* **completion_date**
+* **primary\_completion\_date**
+* **phase** - current phase of the clinical trial (1-4)
+* **study_type** - classification of the study
+	* Values: Expanded Acess, N/A, Other
+* **has\_expanded\_access** - does the clinical trial allow patients with immediately life threatening conditions to participate
+* **study\_design\_info**
+	* **allocation** - how the tests are administered to the patients
+		* Value: Randomized
+	* **intervention_model** - general design of the strategy for assigning treatments to participants
+	* **primary_purpose** - reason for the clinical trial
+		* Values: Treatment
+	* **masking** - tells who knows which people were treated
+* **primary_outcome**
+	* **measure** - important for evaluating the effect of the treatment, what it is supposed to do
+	* **time_frame** - ???
+* **number\_of\_arms** - number of group/sub-groups in the clinical trial
+* **enrollment** - number of participants in the clinical study tagged with a type
+	* Types: Actual, Anticipated, Estimate
+* **condition** - disease, disorder, etc being studied - can be multiple
+* **arm_group** 
+	* **arm\_group\_label**
+	* **arm\_group\_type**
+	* **description**
+* **intervention** - treatment
+	* **intervention_type** - type of treatment
+		* Values: Behavioral, Biological, Combination Product, Device, Diagnostic Test, Dietary Supplement, Drug, Genetic, Procedure, Radiation, Other
```

Comment:
Should "Genetic, Procedure" have a comma in it?

Reply:
Yes, they are two separate values.,